test/price_comparison_utils_test.ts

```diff
@@ -228,88 +227,6 @@ describe(SUITE_NAME, () => {
             ]);
         });
 
-        it('returns the Kyber results with highest makerAmount when quoting sellAmount', () => {
```

Comment:
why is a Kyber test removed if we re-enabled Kyber?

Reply:
Ah this refers to legacy Kyber (which have been deprecated and disabled). 
Updated PR description to clarify.